apps/base-docs/docs/account-abstraction-with-base-paymaster/intro-to-account-abstraction.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Intro to Account Abstraction
+description: Learn what Account Abstraction it, how it works and what problems it solves.
+hide_table_of_contents: false
+---
+
+ERC-4337, also known as "Account Abstraction", is a standard that allows smart contracts to initiate transactions, thus enabling any logic that users want to implement to be encoded into the smart contract wallet itself for execution on Ethereum.
+
+Account Abstraction has the potential to be a massive game changer for onchain user experience and many believe it will play a key role in bringing the next billion users onchain.
+
+---
+
+## Prerequisites
```

Comment:
I've moved prereqs to the overview doc so that it's not repeated 4 docs in a row:  https://github.com/base-org/web/blob/master/apps/base-docs/docs/connecting-to-the-blockchain/overview.md